src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -281,9 +281,13 @@ jint ShenandoahHeap::initialize() {
   }
 
   // Reserve aux bitmap for use in object_iterate(). We don't commit it here.
-  ReservedSpace aux_bitmap(_bitmap_size, bitmap_page_size);
+  // Note: on Linux, in THP "advise" mode, we refrain from advising the system to use large pages
+  //  since we know these commits will be short lived.
+  const size_t aux_bitmap_page_size =
+    LINUX_ONLY(UseTransparentHugePages ? os::vm_page_size() :) bitmap_page_size;
```

Comment:
There is a `ifdef LINUX` THP-related block in the same file, so better to match its style:

```
   size_t aux_bitmap_page_size = bitmap_page_size;
#ifdef LINUX
   // In THP "advise" mode, we refrain from advising the system to use large pages
   // since we know these commits will be short lived, and there is no reason to trash
   // the THP area with this bitmap.
   if (UseTransparedHugePages) {
     aux_bitmap_page_size = os::vm_page_size();
   }
#endif 
```